king_phisher/client/application.py

```diff
@@ -411,7 +437,7 @@ def theme_file(self):
 			return DISABLED
 		return find.find_data_file(os.path.join('style', self._theme_file))
 
-	def load_config(self, load_defaults=False):
+	def do_config_load(self, load_defaults=False):
```

Comment:
Since this is a signal handler, `load_defaults` should no longer be a kwarg, it should be required.